avocado/utils/kernel.py

```diff
@@ -72,7 +72,8 @@ def download(self):
         full_url = self.URL + self.SOURCE.format(version=self.version)
         self.asset_path = asset.Asset(full_url, asset_hash=None,
                                       algorithm=None, locations=None,
-                                      cache_dirs=self.data_dirs).fetch()
+                                      cache_dirs=[self.work_dir], expire=0).fetch()
```

Comment:
The point is that `work_dir` (as a cache_dir) is going to be a new one anytime, anyway.  Actually, I was wondering if it's possible to have a "library default" cache dir, that could be used as a default value for `cache_dirs`.